### PR TITLE
Changed from NymN reddit to forsen reddit

### PR DESCRIPTION
**Pull request by mynamecheff**

### CODE REVIEW

File: commands/Reddit.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  permission: 100,
+  category: 'Random command',
+  execute: async (channel, user, input, perm) => {
+    try {
+      if (module.exports.permission > perm) {
+        return;
+      }
+
+      const response = await got('https://www.reddit.com/r/RedditAndChill/.json', { json: true });
+      const posts = response.body.data.children;
```
Comment (JoachimFlottorp) [inline commands/Reddit.js:16]: this should just be response.data.children

File: commands/Reddit.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  description: 'This command will list the top 3 posts on the Forsen subreddit.',
+  permission: 100,
+  category: 'Random command',
+  execute: async (channel, user, input, perm) => {
+    try {
+      if (module.exports.permission > perm) {
+        return;
+      }
+
+      const response = await got('https://www.reddit.com/r/RedditAndChill/.json', { json: true });
```
Comment (JoachimFlottorp) [inline commands/Reddit.js:15]: Change this from got(..., { json: true })

To got(...).json();

File: commands/Reddit.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      const posts = response.body.data.children;
+
+      let top3 = [];
+      for (let i = 0; i < 3; i++) {
+        top3.push(`${i+1}. ${posts[i].data.title} - by ${posts[i].data.author}`);
+      }
+
+      return top3.join('\n');
```
Comment (JoachimFlottorp) [inline commands/Reddit.js:23]: Twitch doesn't allow newlines.

**Review by JoachimFlottorp — CHANGES_REQUESTED** [review]

File: commands/Reddit.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+const { got } = require('./../got');
+/* lists top 3 posts from NymN reddit */
+module.exports = {
+  name: 'Reddit',
```
Comment (hotbear1110) [inline commands/Reddit.js]: I would like this to be lowercase, same with the filename

File: commands/Reddit.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
+const { got } = require('./../got');
+/* lists top 3 posts from NymN reddit */
+module.exports = {
+  name: 'Reddit',
+  ping: true,
+  description: 'This command will list the top 3 posts on the Forsen subreddit.',
```
Comment (hotbear1110) [inline commands/Reddit.js]: This still says forsen

**Comment by hotbear1110** [comment]
If possible, links to the posts would maybe be nice to have

**Review by hotbear1110 — APPROVED** [review]
seemsgood

**Pull request merged**